github-pull-requests.go

```diff
@@ -92,9 +91,7 @@ func populatePRs(rName string, url string) {
 		for _, assignee := range elem.Assignees {
 			eachPRIssue.Assignees += assignee.Login + " "
 		}
-		eachPRIssue.UpdatedAt = elem.UpdatedAt.Format(buzzTimeLayout)
-		delta := elem.UpdatedAt.Sub(elem.CreatedAt)
-		eachPRIssue.Hours = int64(delta.Hours())
+		eachPRIssue.UpdatedAt = elem.UpdatedAt.Unix()
```

Comment:
Yes — instead of doing it in Go it's done in the [browser](https://github.com/brendanashworth/buzz/blob/797abe4c8f55f8b2bd15bc8270a5789bf623f786/static/js/functions.js#L106) now. You may have noticed that the browser doesn't correctly sort by age, this is the reason. It needs a number to sort by, otherwise it sorts by alphabetical date ("Feb" before "Jan"), which isn't relevant.